CMakeLists.txt

```diff
@@ -131,6 +131,18 @@ message(STATUS "C++ Standard: ${CMAKE_CXX_STANDARD}")
 message(STATUS "C++ Standard Required: ${CMAKE_CXX_STANDARD_REQUIRED}")
 message(STATUS "C++ Extensions: ${CMAKE_CXX_EXTENSIONS}")
 
+# find Linux Distribution name
+IF(CMAKE_SYSTEM_NAME MATCHES "Linux")
+  execute_process (
+          COMMAND bash -c "awk -F= '/^ID=/{print $2}' /etc/os-release |tr -d '\n' | tr -d '\"'"
+          OUTPUT_VARIABLE LinuxOS
+  )
+ELSE()
+  set(LinuxOS "n/a")
+ENDIF()
+
+MESSAGE(STATUS "Linux Distro: ${LinuxOS}")
+
```

Comment:
This isn't doing anything for the linking issue, might be better to remove?

Reply:
done, just removed it. I thought it could be interesting to get this info, but indeed no added value so I removed it